Expensify/App
# PR 1: Moving EmojiPicker as an Independent Component

### Details

- This PR also moves EmojiPicker Popup as an independent comment that can be used by any text input component.

Because the PR is large enough and I am currently working on making EmojiPicker into a singleton component, I am raising this PR that moves EmojiPicker as an independent component.

### Fixed Issues

$ https://github.com/Expensify/App/issues/3245

### Tests
1. Tested that Emojis work fine as before
2. Tested the skintone selection for Compose
3. Tested the frequently used emojis for Compose



### QA Steps

1. Go to any chat
2. Click on EmojiPicker
3. Select the Emoji and enter
4. You can change the Skintone and ensure that it works fine as before.


### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/3069065/152121297-c62bd5bf-f721-474e-a493-160f036e2705.mov

#### Mobile Web

https://user-images.githubusercontent.com/3069065/152123616-8204a4af-5261-4fdb-a137-90e4ba3fdce5.mov

#### Desktop

https://user-images.githubusercontent.com/3069065/152121553-c7958e3a-233c-4766-8568-951ed98fc204.mov

#### iOS

https://user-images.githubusercontent.com/3069065/152126193-dbd5cd05-66e3-438c-9d06-cb1c5a227a49.mov


#### Android

https://user-images.githubusercontent.com/3069065/152123682-ba2b6191-6195-4fd8-846e-1b2e1be314f4.mov
